specification/cognitiveservices/data-plane/ComputerVision/stable/v3.2/examples/SuccessfulDescribeImageWithStream.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
+    "Endpoint": "https://westus.api.cognitive.microsoft.com",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "maxCandidates": 1,
+    "Image": "{binary}"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "description": {
+          "tags": [
+            "person",
+            "man",
+            "outdoor",
+            "window",
+            "glasses"
+          ],
+          "captions": [
+            {
+              "text": "Satya Nadella sitting on a bench",
+              "confidence": 0.48293603002174407
+            },
+            {
```

Comment:
I think we probably want to remove those multiple candidates coming from VLP (just like other places).

Reply:
This is also good catch. Since maxCandidate is set to 1, API should return only one caption. Fixed.